roaringbitmap/src/main/java/org/roaringbitmap/ParallelAggregation.java

```diff
@@ -139,12 +141,8 @@ public ContainerCollector(Function<List<Container>, Container> reducer) {
       RoaringArray ra = bitmap.highLowContainer;
       for (int i = 0; i < ra.size; ++i) {
         Container container = ra.values[i];
-        Short key = ra.keys[i];
-        List<Container> slice = grouped.get(key);
-        if (null == slice) {
-          slice = new ArrayList<>();
-          grouped.put(key, slice);
-        }
+        short key = ra.keys[i];
+        List<Container> slice = grouped.computeIfAbsent(key, k -> new ArrayList<>());
```

Comment:
Please benchmark this. The code was written this way for a reason.

Reply:
The benchmarks proved that you're right. 
I wonder why just even change `Short -> short` has small degradation effect on all benchmarks. Do you know the answer?

Reply:
Autoboxing.

Reply:
Moreover, why edit for the hell of it, if you abstain from all welcome code review?

Reply:
Yup, auto boxing as it is later used as a key in collection, didn't notice that.
D